packages/v4/src/pages/home.js

```diff
@@ -26,8 +26,8 @@ import './home.css';
 const aboutPatternFly = `Hello, we're PatternFly.`;
 const patternFlyPrinciples = 'Flyer fundamentals';
 
-export const HomePage = ({ location, layoutOptions }) => (
-  <SideNavLayout location={location} hasGdprBanner {...layoutOptions}>
+export default ({ location, layoutOptions, children }) => (
```

Comment:
It looks like `children` is destructured but never used?

Reply:
Good call! I played around with it at one point.